tomviz/MainWindow.cxx

```diff
@@ -199,37 +199,32 @@ MainWindow::MainWindow(QWidget* _parent, Qt::WindowFlags _flags)
 
   // Build Tomography menu
   // ################################################################
-
   QAction *toggleDataTypeAction = ui.menuTomography->addAction("Toggle Data Type");
   ui.menuTomography->addSeparator();
 
   QAction *setTiltAnglesAction = ui.menuTomography->addAction("Set Tilt Angles");
   ui.menuTomography->addSeparator();
-    
-  QAction *generateTiltSeriesAction = ui.menuTomography->addAction("Generate Tilt Series");
-  ui.menuTomography->addSeparator();
 
   QAction *subtractBackgroundAction = ui.menuTomography->addAction("Background Subtraction (Manual)");
-  QAction *alignAction = ui.menuTomography->addAction("Translation Align");
-  QAction *autoAlignAction = ui.menuTomography->addAction("Translation Align (Auto)");
-  QAction *rotateAlignAction = ui.menuTomography->addAction("Rotation Align");
+  QAction *alignAction = ui.menuTomography->addAction("Tilt Series Registration (Manual)");
```

Comment:
@cryos @yijiang1 I suggested "Tilt Series Registration" mostly because I don't like "Translation Alignment", since the "Rotation Axis Alignment" also does translation.  I agree that calling it registration may be confusing, and is not standard for us either.

Perhaps a better option would be to just call it "Image Alignment" to clarify that it is operating on images individually rather than the stack as a whole, which "Rotation Axis Alignment" does.  @Hovden what do you think?